metis-harvesting/src/main/java/eu/europeana/metis/harvesting/http/HttpHarvesterImpl.java

```diff
@@ -206,6 +188,17 @@ public FileIterator(Path extractedDirectory) {
       this.extractedDirectory = extractedDirectory;
     }
 
+    @Override
+    public void deleteIteratorContent() {
+      if (extractedDirectory != null) {
+        try {
+          FileUtils.deleteDirectory(extractedDirectory.toFile());
+        } catch (IOException e) {
+          LOGGER.warn("Could not delete directory.", e);
+        }
+      }
+    }
```

Comment:
add logger when ```extractedDirectory``` is ```null```

Reply:
I don't see why that would be useful here.

Reply:
It is the feedback that nothing was removed when method was called.

Reply:
I understand, but why would we care to know if there was nothing to remove? :D
Also normally there should be something to remove if someone is using this method.

Reply:
In case of reading logs we can detect why the things are not being deleted, and  we can determine that this path of code is being followed. perhaps ```LOGGER.debug or warn("Nothing has been deleted")``` I had this idea when we read logs later that is the reason why I think.